netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
```

Comment:
i tried to explain this to you several times how it works

some googling for you:
http://rpm.org/api/4.4.2.2/config_macros.html
https://en.opensuse.org/openSUSE:RPM_conditional_builds
https://bugzilla.redhat.com/show_bug.cgi?id=112663

what you see, is that it undefines it on platforms that do not support it. uncondtionally what is on commandline. that is what you wanted. and there's no point enabling it if support is not there.